eng/pipelines/runtimelab/install-wasi-sdk.cmd

```diff
@@ -0,0 +1,16 @@
+mkdir "%1" 2>nul
+cd /D "%1"
+
+echo Installing Wasi SDK
+
+powershell -NoProfile -NoLogo -ExecutionPolicy ByPass -File "%~dp0install-wasi-sdk.ps1"
+if %errorlevel% NEQ 0 goto fail
+
+echo Setting WASI_SDK_PATH to %1\wasi-sdk-20.0+m
+echo ##vso[task.setvariable variable=WASI_SDK_PATH]%1\wasi-sdk-20.0+m
```

Comment:
> `echo ##vso[task.setvariable variable=WASI_SDK_PATH]%1\wasi-sdk-20.0+m`

Would it be possible to not hardcode the version (not that we should go out of our way in doing so, but it'd be nice to have one place to update in CI when we upgrade).

Reply:
yes, ok.  Have `mv`ed the version named folder to just `wasi-sdk`  It's 2 places, but in the same file now.